content/language/hoon/guides/README.md

```diff
@@ -12,3 +12,4 @@ Guides to various features of the Hoon language.
 - [vases](vases.md) - dynamic data types in hoon.
 - [Sets](sets.md) - Explore set relations using the Hoon standard library.
 - [Implementing an Aura](aura-guide.md) - How to implement support for a new in Hoon.
+- [Nested Core Pattern](nested-core-pattern.md) - Introduction to the nested core pattern.
```

Comment:
Not sure why this shows up as an external link to `https://github.com/urbit/docs.urbit.org/blob/gitbook/content/language/hoon/guides/nested-core-pattern.md`, since it's the same format as other links here. Could it be because this page isn't published in the `gitbook` branch?

Reply:
Fixed; new content has to be added to `SUMMARY.md`.